code/modules/mob/living/carbon/human/species/station/skeleton.dm

```diff
@@ -0,0 +1,63 @@
+/datum/species/skeleton
+	name = SPECIES_SKELETON
+	name_plural = "skeletal remains"
+	blurb = "A human of the skeletal variety."	// Could use a neat OS lore bit here
```

Comment:
re use the remains description for now
"They look like human remains. Some poor soul expired here, a million miles from home."

Reply:
Null Sector is our solar system, right? If so, we should adjust the last part.

Reply:
I guess that also depends on the people's perspective of Earth. As far as I know, nobody alive has even seen it. I'll leave the "million miles from home" bit.